web-ui/src/main/resources/catalog/components/search/resultsview/partials/viewtemplates/editor.html

```diff
@@ -15,7 +15,7 @@
         <div class="col-lg-8 col-md-8">
           <a href=""
              data-ng-show="md.isTemplate != 's'"
-             data-ng-href="catalog.search#/metadata/{{md.getUuid()}}"
+             data-ng-href="catalog.search#/{{(md.draft == 'y')?'metadraf':'metadata'}}/{{md.getUuid()}}"
```

Comment:
Like this?
![imagen](https://user-images.githubusercontent.com/726590/56907294-39e4fd80-6aa4-11e9-935e-f04f758601a0.png)

Reply:
Hydrological Basins is published and approved but no draft (still working on the bug above) while the first two metadata have a working copy (draft, I can change that string) and there you can see the validation status of each (which is the same as the published copy on this case, but could be different).

Reply:
On the dashboard editor there is no row for draft to avoid confusion. You can see there is a Working copy/draft there, and if you try to edit that row, you are redirected to the draft.

Reply:
(I'm trying to get rid of the string "draft" when not referring to the "Draft" status)

Reply:
Maybe @MichelGabriel would have suggestion ?